src/containers/Comments/CommentBodyContainer.jsx

```diff
@@ -0,0 +1,94 @@
+import React, {Component} from "react";
+import CKEditor from '@ckeditor/ckeditor5-react';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import {Button} from "semantic-ui-react";
+import PropTypes from "prop-types";
+import {SUCCEEDED} from "../../store/actions/async";
+
+const commentBoxConfigEdit = {
+    toolbar: ['bold', 'italic', '|', 'bulletedList', 'numberedList', 'blockQuote', '|', 'undo', 'redo'],
+    placeholder: 'Update comment...'
+};
+
+class CommentBodyContainer extends Component {
+    constructor(props) {
```

Comment:
The `constructor` is unnecessary.
`this.state = {...}` is enough.

Reply:
In this situation, I am making use of a `prop` to `initialize` the `state`, so I think it is still necessary.

Reply:
You can access `props`  without the constructor.